### PR TITLE
Redirect nonexistent reference pages to app-services docs

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
### Jira

- issue reported [on slack](https://mongodb.slack.com/archives/CLY6HQWUE/p1656568338864189)

### Staged Changes (Requires MongoDB Corp SSO)

- autobuilder cannot preview redirect changes, unfortunately

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: config/redirects [hunk 1]
```diff
@@ -756,4 +756,14 @@ raw: ${prefix}/sdk/dotnet/advanced-guides/link-user-identities -> ${base}/sdk/do
 raw: ${prefix}/sdk/dotnet/advanced-guides/multi-user-applications -> ${base}/sdk/dotnet/examples/users/multi-user-applications
 raw: ${prefix}/sdk/dotnet/examples/authenticate -> ${base}/sdk/dotnet/examples/users/authenticate
 raw: ${prefix}/sdk/dotnet/examples/manage-email-password-users -> ${base}/sdk/dotnet/examples/users/manage-email-password-users
-raw: ${prefix}/sdk/dotnet/examples/manage-user-api-keys -> ${base}/sdk/dotnet/examples/users/manage-user-api-keys
\ No newline at end of file
+raw: ${prefix}/sdk/dotnet/examples/manage-user-api-keys -> ${base}/sdk/dotnet/examples/users/manage-user-api-keys
+
+# Redirect reference section to atlas/app-services
+raw: ${prefix}/reference/find-your-project-or-app-id> -> ${appServicesBase}/reference/find-your-project-or-app-id>
```
[the remark this answers is not in the document]
Comment (nathan-contino-mongo) [reply config/redirects:762]: d'oh, thank you. This is why I wish we could test these.

File: config/redirects [hunk 2]
```diff
@@ -756,4 +756,14 @@ raw: ${prefix}/sdk/dotnet/advanced-guides/link-user-identities -> ${base}/sdk/do
...
+raw: ${prefix}/reference/authenticate-http-client-requests> -> ${appServicesBase}/reference/authenticate-http-client-requests>
+raw: ${prefix}/reference/issues-and-workarounds> -> ${appServicesBase}/reference/issues-and-workarounds>
+raw: ${prefix}/reference/third-party-licenses> -> ${appServicesBase}/reference/third-party-licenses>
+raw: ${prefix}/reference/upgrade-shared-cluster> -> ${appServicesBase}/reference/upgrade-shared-cluster>
+raw: ${prefix}/reference/service-limitations> -> ${appServicesBase}/reference/service-limitations>
+raw: ${prefix}/reference/push-notifications> -> ${appServicesBase}/reference/push-notifications>
+raw: ${prefix}/reference/services> -> ${appServicesBase}/reference/services>
```
Comment (nathan-contino-mongo) [inline config/redirects:769]:
```suggestion
raw: ${prefix}/reference/authenticate-http-client-requests -> ${appServicesBase}/reference/authenticate-http-client-requests/
raw: ${prefix}/reference/issues-and-workarounds -> ${appServicesBase}/reference/issues-and-workarounds/
raw: ${prefix}/reference/third-party-licenses -> ${appServicesBase}/reference/third-party-licenses/
raw: ${prefix}/reference/upgrade-shared-cluster -> ${appServicesBase}/reference/upgrade-shared-cluster/
raw: ${prefix}/reference/service-limitations -> ${appServicesBase}/reference/service-limitations/
raw: ${prefix}/reference/push-notifications -> ${appServicesBase}/reference/push-notifications/
raw: ${prefix}/reference/services -> ${appServicesBase}/reference/services/
```

File: config/redirects (hunk 1 above)
Comment (nathan-contino-mongo) [inline config/redirects:762]:
```suggestion
raw: ${prefix}/reference/find-your-project-or-app-id -> ${appServicesBase}/reference/find-your-project-or-app-id/
```

**Review by dacharyc — APPROVED** [review]
Nice, looking good!

**Pull request merged**